integration-tests/features/importing_coverage.feature

```diff
@@ -25,82 +25,82 @@ Feature: Importing coverage data
               """
           AND the following metrics have following values:
               | metric                  | value |
-              | coverage                | 23.8  |
-              | line_coverage           | 17.6  |
+              | coverage                | 20.0  |
```

Comment:
well :) it does not make any sense to me at all. i get 100 coverage of a file that is fully covered and thats ok. but then that 100 % is set to all project because unit tests are not taken into account and execution lines metrics is not set. ive checked our own projects that use bullses (so there is cov information to everyfile) and the results there are comparable with slight differences likely due to unit tests or somehitng.


anyway we need to stabilize the coverage feature once this is merged properly. and create updated coverage projects that currently we dont have since the  samples and coverage files do not match